code/controllers/subsystem/profiler.dm

```diff
@@ -0,0 +1,48 @@
+SUBSYSTEM_DEF(profiler)
+	name = "Profiler"
+	init_order = INIT_ORDER_PROFILER
+	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
+	wait = 3000
+	flags = SS_NO_TICK_CHECK
+	var/fetch_cost = 0
+	var/write_cost = 0
+
+/datum/controller/subsystem/profiler/stat_entry()
+	..("F:[round(fetch_cost,1)]ms | W:[round(write_cost,1)]ms")
+
+/datum/controller/subsystem/profiler/Initialize()
+	if(!config.auto_profile)
+		StopProfiling() //Stop the early start profiler if we dont want it on in the config
```

Comment:
Shouldn't it get the `SS_NO_FIRE` flag to avoid useless calls?
Or just the `can_fire` value being set to `FALSE` if you do want to be able to enable this SS more easily midgame